Support install dependencies from vcpkg

shrinktofit opened the pull request:
The connan PM  usually failed to work (at least for me) on Windows. By comparison, the vcpkg is commonly used.

zellski commented:
I see you closed this PR - did you change your mind?

I was trying to figure out how I felt about it, and I wanted to ask in what way Conan was giving you trouble. I wasn't crazy about making CMakeLists.txt so complex. On the other hand, a build system's job is to make building easy.

shrinktofit commented:
> I see you closed this PR - did you change your mind?
> 
> I was trying to figure out how I felt about it, and I wanted to ask in what way Conan was giving you trouble. I wasn't crazy about making CMakeLists.txt so complex. On the other hand, a build system's job is to make building easy.

When I run
```bash
> conan install . -i build -s build_type=Debug
```
I got
```
ERROR: Missing prebuilt package for 'boost_system/1.69.0@bincrafters/stable'
Try to build it from sources with "--build boost_system"
Or read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package"
```

So I changed to run
```bash
> conan install . -i build -s build_type=Debug --build boost_system
```
I got
```
boost_system/1.69.0@bincrafters/stable:
boost_system/1.69.0@bincrafters/stable: ERROR: Package 'a7dd34237987889dcbaf7fd1429e3130232d8f63' build failed
boost_system/1.69.0@bincrafters/stable: WARN: Build folder C:\.conan\0e7f71\1
ERROR: boost_system/1.69.0@bincrafters/stable: Error in build() method, line 147
        self._build_common()
while calling '_build_common', line 170
        self.run(" ".join(b2_command))
        ConanException: Error 1 while executing b2 -j4 -d+1 -a --hash=yes --debug-configuration --layout=system    system-build
```

-------------------

I'm not familiar with conan So I changed to use vcpkg, which is popular on Windows. But I doubt that only me encounter this problem so I close the PR for now :)

zellski commented:
Yeah, building boost from source locally is not likely to succeed. And whether or not a precompiled version exists for any particular build setup, that seems to somewhat randomly depend on whoever set up that library's build. I've found it **very** difficult to figure out exactly what support matrix to expect for different libraries.

I suspect you could get it to work by adding a bunch of `-s compiler.version` type arguments to the command line, rather than allow Conan to inspect your system. Perhaps I'll experiment a bit with that and update the documentation.

shrinktofit commented:
That's cool! 
When I use
```bash
> conan install . -i build -s build_type=Debug -s compiler="Visual Studio" -s compiler.version
=15
> conan build . -bf build
```
it works. (I use Visual Studio 2019)
You're the man! The README could be updated so.

shrinktofit commented:
By the way, the FBX SDK could be setup as this PR suggests.

shrinktofit commented:
> By the way, the FBX SDK could be setup as this PR suggests.

> By default, FBX2glTF will look into `<FBX2glTF-root-dir>/sdk/<your-platform>/<FBX-SDK-version>`.
> Usually you didn't install the FBX SDK into that directory.
> So simply create a symbolic link to the real FBX SDK here.
> 
> The following scripts is powershell scripts.
> 
> First you should create the `sdk` directory if it doesn't exist. cd it after all.
> 
> ```ps1
> > mkdir sdk
> > cd sdk
> ```
> 
> Now create the symbolic link. (You possibly need the admin privilege)
> 
> ```ps1
> > New-Item -Path Windows -ItemType SymbolicLink -Value "path-to-fbx-sdk"
> ```
> 
> the `path-to-fbx-sdk` should be something like "C:\Program Files\Autodesk\FBX\FBX SDK".
